doc/development/migration_guides.rst

```diff
@@ -0,0 +1,27 @@
+Migration guides
+================
+
+The migration guides below offers guidance for dealing with major breaking
+changes between Strawberry Fields releases.
+
+Version 0.20.0
+--------------
+
+Command Line Interface
+^^^^^^^^^^^^^^^^^^^^^^
+
+The following table shows the equivalent ``xcc`` (v0.20.0) command for
+each ``sf`` (v0.19.0) command:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - **Strawberry Fields v0.19.0**
+     - **Strawberry Fields v0.20.0**
+   * - ``sf configure --token "foo"``
+     - ``xcc config set REFRESH_TOKEN "foo"``
+   * - ``sf --ping``
+     - ``xcc ping``
+   * - ``sf run "foo.xbb"``
+     - ``xcc job submit --name "bar" --target "X8_01" --circuit "$(cat foo.xbb)"``
```

Comment:
This might be worth duplicating in the changelog 🙂 

In terms of communicating breaking changes, I'm very much a fan of 'overcommunication'. Especially since the changelog doesn't need to be maintained!

Reply:
(table renders really nicely btw)

Reply:
A sudden minor thought, but `--circuit "$(cat foo.xbb)"` would be very unix specific -- what should windows users using `xcc` do here?

Reply:
Thanks for pointing all of this out, @josh146 and @thisac!

> I wonder if this might be worth duplicating in the changelog 🙂

Done. Unfortunately, the MD → RST converter does not like tables so I've resorted to a list format instead.

> A sudden minor thought, but `--circuit "$(cat foo.xbb)"` would be very unix specific -- what should windows users using `xcc` do here?

Ouch, I hadn't considered that! It turns out that you can run the following in place of `cat foo.xbb` in PowerShell:
```powershell
Get-Content foo.xbb -Raw
```

Reply:
> Ouch, I hadn't considered that! It turns out that you can run the following in place of cat foo.xbb in PowerShell:

I'm not sure if submitting files from the terminal is a common workflow, but if it is, it might make sense to include the file read as part of the `xcc` library, since it can then be written in Python/be platform independent?

If it is not a common workflow, we could probably just remove it from the docs altogether.

Reply:
> I'm not sure if submitting files from the terminal is a common workflow, but if it is, it might make sense to include the file read as part of the `xcc` library, since it can then be written in Python/be platform independent?

Yeah, in hindsight, I think you're right. We should implement this feature in the following XCC release.